package.json

```diff
@@ -64,11 +69,22 @@
   "browserslist": [
     "defaults"
   ],
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm run husky_msg && npm run compile_css",
+      "pre-push": "npm run husky_msg && npm run update_browserslist"
+    }
+  },
   "scripts": {
     "bower": "bower install",
+    "husky_msg": "echo Running git hooks -- see the 'husky' section of package.json for details",
+    "black": "black -v --exclude '.+Impl.py' .",
+    "flake8": "flake8 .",
     "test": "grunt test",
     "update_browserslist": "npx browserslist@latest --update-db",
     "compile_css": "sass kbase-extension/scss/:kbase-extension/static/kbase/css/ && grunt postcss:concat",
+    "watch_css": "grunt watch",
+    "stylelint": "stylelint --config .stylelintrc.yaml --fix kbase-extension/scss/**/*.scss",
```

Comment:
check scss files for errors, automatically fixing those that it is possible to fix